YoCode/Program.cs

```diff
@@ -1,23 +1,53 @@
 ﻿using System;
+using System.Linq;
 using System.IO;
 
 namespace YoCode
 {
     public class Program
     {
+
         static void Main(string[] args)
         {
+            var modifiedTestDirPath = args[0];
+            var originalTestDirPath = args[1];
+
+            var consoleOutput = new PrintToConsole();
```

Comment:
👍 on having a type for outputting messages. This will make it easier for you to have your tests write to, e.g. a `StringBuilder`, so you can check that the output is correct.